apps/studio/src/components/Datatable/Datatable.tsx

```diff
@@ -136,18 +136,11 @@ export const Datatable = <T extends object>({
         </Table>
       </Box>
       <Flex py="1rem" gap="1rem">
-        {totalRowCountString && (
-          <Text textStyle="caption-2" color="base.content.medium">
-            {totalRowCountString}
-          </Text>
-        )}
         {pagination && (
```

Comment:
this is the one that may need to change. Only render if there are rows:

```ts
totalRowCount > 0 && pagination && <...>
```

Reply:
@seaerchin this still requires change, since it may look weird if there are no rows